.github/workflows/update-citation-cff.yaml

```diff
@@ -8,21 +8,17 @@ on:
     branches:
       # This may seem like a no-op but it prevents triggering on tags.
       # We use '**' rather '*' to accomodate names like 'dev/branch-1'
```

Comment:
This now outdated comment should be removed

Reply:
```suggestion
```